developer/src/kmc-ldml/src/compiler/messages.ts

```diff
@@ -7,7 +7,7 @@ const SevError = CompilerErrorSeverity.Error | CompilerErrorNamespace.LdmlKeyboa
 const SevFatal = CompilerErrorSeverity.Fatal | CompilerErrorNamespace.LdmlKeyboardCompiler;
 
 export class CompilerMessages {
-  static Error_InvalidNormalization = (o:{form: string}) => m(this.ERROR_InvalidNormalization, `Invalid normalization form '${o.form}`);
+  static Error_InvalidNormalization = () => m(this.ERROR_InvalidNormalization, `normalization=disabled is not allowed.`);
   static ERROR_InvalidNormalization = SevError | 0x0001;
```

Comment:
I thought Keyman would support normalization=disabled?

Reply:
make it a HINT?

Reply:
I'm not worried about CLDR compliance unitl we have some more policies sorted out.

Reply:
```
static Hint_NormalizationDisabled = () => m(this.HINT_NormalizationDisabled, `normalization=disabled is not recommended.`);
```

?

Reply:
PTAL